csbuild/toolchain/toolchain.py

```diff
@@ -1426,26 +1434,6 @@ def testStaticFunctionCalls(self):
 		mixin2.Derived2Static()
 		self.assertChanged(derived2Static = 1)
 
-	def testRemoveTool(self):
```

Comment:
Curious why this test needed to be removed.

Reply:
It was an invalid test. This function affects project plans so it isn't self contained and thus can't be unit tested independent of project plans. I will add a proper functional test for it later.